tools/offsets-tool-py/Makefile

```diff
@@ -1,7 +1,6 @@
 ACTIVATE=offtool/bin/activate
 
 setup: $(ACTIVATE)
-	. $(ACTIVATE) && pip3 install clang
```

Comment:
is this a temporary thing? if not, we can remove this `virtualenv` stuff.

Reply:
Doesn't look like that repo is getting updated.

Reply:
Will clean up in a follow-up PR.